src/main/ApplicationImpl.cpp

```diff
@@ -802,24 +795,6 @@ ApplicationImpl::validateAndLogConfig()
         }
     }
 
-    if (mConfig.BACKGROUND_EVICTION_SCAN)
-    {
-        if (!mConfig.isUsingBucketListDB())
-        {
-            throw std::invalid_argument(
-                "BACKGROUND_EVICTION_SCAN set to true but "
-                "DEPRECATED_SQL_LEDGER_STATE is set to true. "
-                "DEPRECATED_SQL_LEDGER_STATE must be set to false to enable "
-                "background eviction.");
-        }
-
-        if (mConfig.WORKER_THREADS < 2)
```

Comment:
I removed this check because it's redundant, we already enforce the minimum of two in Config.cpp:1337
```
{"WORKER_THREADS",
 [&]() { WORKER_THREADS = readInt<int>(item, 2, 1000); }},
```